server/src/server.ts

```diff
@@ -51,9 +53,11 @@ app.use(bodyParser.urlencoded({ extended: false }))
 app.use(compression())
 // routes
 app.use('/api/v1/urlcheck', UrlCheckRouter)
+app.use('/api/v1/updateddo', UpdateRouter)
```

Comment:
Maybe go even more simple and just name the endpoints `/update` & `/retire`?

Reply:
We could go for:
POST/PATCH?: '/api/v1/ddo/<did>' for update
DELETE: '/api/v1/ddo/<did>' for retire
Since we don't know what else will be included in this backend

Reply:
yeah, this sounds good!

Reply:
or maybe `PUT` for updating cause that#s what Aquarius API does https://docs.oceanprotocol.com/references/aquarius/#assetsddodid

Reply:
Resolved in 0cb39e3